packages/gestalt/src/Datapoint/DataPointBaseProps.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
```

Comment:
Please don't do this — it makes it more difficult to accurately estimate the scope of disabled lint rules. If suppressions are needed, add them line-by-line.